bundler/spec/install/gems/fund_spec.rb

```diff
@@ -52,6 +52,35 @@
       end
     end
 
+    context "when gems include a fund URI but `ignore_funding_requests` is configured" do
+      let(:config) { "ignore_funding_requests true" }
+
+      before do
+        bundle "config set #{config}" if config
```

Comment:
Why the conditional? `config` should be always defined, right? Can you instead do `bundle "config set ignore_funding_requests true"` directly?

Reply:
Good point, I've changed it now.

I had just copied the first version from a different spec and
left it as is for consistency - but you are right, didn't make
sense for this one as the value is unlikely to ever change.